src/ripple_app/paths/RippleCalc.cpp

```diff
@@ -2519,6 +2519,9 @@ TER RippleCalc::rippleCalc (
         if (tesSUCCESS == pspDirect->terStatus)
            pspDirect->checkNoRipple (uDstAccountID, uSrcAccountID);
 
+        if(pspDirect->terStatus == tesSUCCESS)
+            pspDirect->checkFreeze();
```

Comment:
I don't think you can do this: it looks like it's going to cause all sorts of bad things (like breaking path finding if a frozen account happens to have a good rate)